Automattic/pocket-casts-android
# Profile bookmarks - Sync Sort Option

## Description
Adds profile bookmarks sort option sync support between devices.

## Testing Instructions

- Install develop build variant
- Start the app on two different devices (D1, D2) with the same account that needs to have Plus active
- D1: Go to Profile -> Bookmarks
- D1: Set a sort option
- D1: Go back to the profile screen and tap Refresh now
- D2: Go to the profile screen and tap Refresh now
- D2: Open Profile -> Bookmarks
- D2: Ensure that the sort option active is the same as in the D1 device.

## Checklist
 
#### I have tested any UI changes...
